onnxruntime/core/providers/cpu/math/top_k.cc

```diff
@@ -170,16 +234,47 @@ template <>
 Status TopK<10, float>::Compute(OpKernelContext* p_op_kernel_context) const {
   const auto* X = p_op_kernel_context->Input<Tensor>(0);
   const auto* Y = p_op_kernel_context->Input<Tensor>(1);
-  if (X == nullptr || Y == nullptr) return Status(common::ONNXRUNTIME, common::FAIL,
+  if (X == nullptr || Y == nullptr) return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL,
                                                   "input count mismatch, expected 2 inputs - "
```

Comment:
nit: having a return statement on the same line as a condition makes it very difficult to set a breakpoint on the return being hit

Reply:
Fixed.